tasks/main.yml

```diff
@@ -1,27 +1,82 @@
-# SPDX-License-Identifier: MIT
 ---
-- name: Set platform/version specific variables
-  include_tasks: tasks/set_vars.yml
-
-# Examples of some tasks:
-- name: Ensure required packages are installed
+- name: Ensure sudo is installed
   package:
-    name: "{{ __template_packages }}"
+    name: sudo
     state: present
-    use: "{{ (__template_is_ostree | d(false)) |
-             ternary('ansible.posix.rhel_rpm_ostree', omit) }}"
+  register: sudo_installed
+
+- name: Set include directories variable
+  set_fact:
+    sudo_include_dirs: "{{ sudo_sudoers_files
+    | selectattr('include_directories', 'defined')
+    | map(attribute='include_directories') | flatten }}"
+
+- name: Set sudoer file paths variable
+  set_fact:
+    sudo_file_paths: "{{ sudo_sudoers_files
+    | selectattr('path', 'defined')
+    | map(attribute='path') | flatten }}"
+
+- name: Ensure include directories are created
+  file:
```

Comment:
I wonder if we need this task, I think the below template task would create directories, so no need to create them in a separate task

Reply:
But with different permissions.